src/main/java/uk/gov/ons/census/fwmt/rmadapter/service/impl/JobCacheManagerImpl.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ons.census.fwmt.rmadapter.service.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.rmadapter.config.RedisUtil;
+import uk.gov.ons.census.fwmt.rmadapter.redis.HouseholdRequestEntity;
+import uk.gov.ons.census.fwmt.rmadapter.service.JobCacheManager;
+
+@Slf4j
+@Service
+public class JobCacheManagerImpl implements JobCacheManager {
+
+  @Autowired
+  private RedisUtil<HouseholdRequestEntity> redisUtil;
+
+  @Override
+  public HouseholdRequestEntity cacheCreateHouseholdRequest(HouseholdRequestEntity householdRequestEntity) {
```

Comment:
move this code to HouseholdStore

Reply:
Moved the cache into the HouseholdStore and removed the JobCacheManager and Impl